Makefile

```diff
@@ -290,16 +305,10 @@ generate_cli_commands_docs:
 test_all: # generate_mocks
 	go test -p 1 -count=1 ./...
 
-.PHONY: test_all_with_json
-## Run all go unit tests, output results in json file
-test_all_with_json: generate_rpc_openapi # generate_mocks
-	go test -p 1 -json ./... > test_results.json
-
-.PHONY: test_all_with_coverage
-## Run all go unit tests, output results & coverage into files
-test_all_with_coverage: generate_rpc_openapi # generate_mocks
-	go test -p 1 -v ./... -covermode=count -coverprofile=coverage.out
-	go tool cover -func=coverage.out -o=coverage.out
+.PHONY: test_all_with_json_coverage
+## Run all go unit tests, output results & coverage into json & coverage files
+test_all_with_json_coverage: generate_rpc_openapi # generate_mocks
+	go test -p 1 -json ./... -covermode=count -coverprofile=coverage.out -run TestUtilityContext_HandleMessage | tee test_results.json | jq
```

Comment:
It looks like this offending line is still here, please clarify if that's intended and if it is that we have a TODO/ticket to track the effort necessary to restore the "all" part of `test_all_with_json_coverage`

Reply:
Thanks for catching this. I reduce the scope of the tests to iterate faster.

Removed and re-running now.